packages/material-ui/src/InputBase/Textarea.test.js

```diff
@@ -136,8 +144,9 @@ describe('<Textarea />', () => {
         getComputedStyle: {
           'box-sizing': 'content-box',
           'padding-top': `${padding}px`,
+          'line-height': '15px',
         },
-        scrollHeight: 30,
+        scrollHeight: 35, // 2 lines + padding
```

Comment:
Using the Volkswagen method for testing here. :car: :dash: But the previous scrollHeight mock was wrong; scrollHeight includes the padding, so it's 5px paddingTop + 2 lines à 15px = 35px.